### PR TITLE
Feature/scv 60+update stac version number

### PR DESCRIPTION
**Pull request by fgabelmannjr**
This PR updates the STAC_VERSION number throughout the application to the latest v1.0.0-b1

### CODE REVIEW

File: docs/development.md [hunk 1]
```diff
@@ -5,53 +5,61 @@
...
-When `yamlUpdater.js` is ran, it looks at the repositories listed below to get the latest version of STAC and WFS3 specification schemas. It then updates the current schema file `OAcore+STAC.yaml` located in the `docs` directory.
+When `yamlUpdater.js` is ran, it looks at the repositories listed below to get the latest version of STAC and WFS3 specification schemas. It then updates the current schema file `WFS3core+STAC.yaml` located in the `docs` directory.
 
 This process is used to ensure that this app is always using the most recent version of `STAC` it is highly recommended that after an update has happened that all of the test are ran to ensure nothing has broken. We also recommend that test are written for any new `STAC` specifications.
 
-STAC: https://github.com/radiantearth/stac-spec/blob/master/api-spec/STAC.yaml
+STAC: <https://github.com/radiantearth/stac-spec/blob/master/api-spec/STAC.yaml>
 
-OA: https://github.com/radiantearth/stac-spec/blob/master/api-spec/openapi/OAFeat.yaml
+WFS3: <https://github.com/radiantearth/stac-spec/blob/master/api-spec/openapi/WFS3.yaml>
```
Comment (DrewE84) [inline docs/development.md:61]: I believe we wanted to keep the OAFeat link. The WFS3 repo no longer exists.

File: docs/development.md [hunk 2]
```diff
@@ -5,53 +5,61 @@
...
 ### Validator
+
 The `validator` directory holds logic to retrieve component schemas from the STAC and WFS3 specification document, and validate component objects against them.
 
 ## Scripts
+
 The `scripts` directory currently contains only one file: `yamlUpdater.js`. This script is ran automatically during the build process, and is intended to update the STAC and WFS3 schema yaml to the current version made accessible by Radiant Earth.
 
-When `yamlUpdater.js` is ran, it looks at the repositories listed below to get the latest version of STAC and WFS3 specification schemas. It then updates the current schema file `OAcore+STAC.yaml` located in the `docs` directory.
+When `yamlUpdater.js` is ran, it looks at the repositories listed below to get the latest version of STAC and WFS3 specification schemas. It then updates the current schema file `WFS3core+STAC.yaml` located in the `docs` directory.
```
Comment (DrewE84) [inline docs/development.md:55]: Similar to my other comment, I think we need to keep the OAcore yaml. If I'm remembering correctly, the people in charge of it either changed the name to OAFeat/OAcore, or they chagned where they're storing the spec yaml.

File: docs/development.md (hunk 1 above)
Comment (fgabelmannjr) [reply docs/development.md:61]: This is an odd change then @DrewE84 .  I need to spend some time digging on this but something is automatically touching files or there is drift in some remotes somehow.  I'll revert this for now however.  Good catch!

File: docs/development.md (hunk 2 above)
Comment (fgabelmannjr) [reply docs/development.md]: Fixed and pushed.

**Comment by fgabelmannjr** [comment]
@DrewE84 This is ready for another look.

**Pull request merged**